hello/patch/README.md

```diff
@@ -0,0 +1,9 @@
+# Patch Workflow Sample
```

Comment:
Sorry to keep being pedantic, but multi-file examples aren't really good for "hello". Can you make this a top-level `patching` example with starter, worker, and these files and what not? Otherwise, come back to single file.

Reply:
re-implemented it with args to run either v1 or v2 of the workflow